tendrl/common/flows/base_flow.py

```diff
@@ -0,0 +1,11 @@
+import abc
+
+
+class BaseFlow(object):
+    __metaclass__ = abc.ABCMeta
+
+    @abc.abstractmethod
+    def run(self):
+        raise NotImplementedError(
```

Comment:
Inherit "NotImplementedError" and create "FlowNotImplementedError" (common/flows/exceptions.py) and provide msg "run function not implemented"